go.mod

```diff
@@ -7,33 +7,34 @@ require (
 	github.com/go-openapi/spec v0.19.3
 	github.com/magiconair/properties v1.8.1 // indirect
 	github.com/mikefarah/yq/v3 v3.0.0-20200615114226-086f0ec6b9aa
-	github.com/operator-framework/operator-sdk v0.19.0
+	github.com/onsi/ginkgo v1.12.1
+	github.com/onsi/gomega v1.10.1
 	github.com/prometheus/client_golang v1.5.1
-	github.com/prometheus/common v0.9.1
-	github.com/spf13/pflag v1.0.5
+	github.com/spf13/pflag v1.0.5 // indirect
 	github.com/stretchr/testify v1.5.1
-	k8s.io/api v0.18.2
-	k8s.io/apimachinery v0.18.2
+	go.uber.org/zap v1.10.0
+	k8s.io/api v0.18.6
+	k8s.io/apimachinery v0.18.6
 	k8s.io/client-go v12.0.0+incompatible
 	k8s.io/code-generator v0.18.2
 	k8s.io/gengo v0.0.0-20200114144118-36b2048a9120
 	k8s.io/heapster v1.5.4 // indirect
 	k8s.io/kube-aggregator v0.0.0
-	k8s.io/kube-openapi v0.0.0-20200121204235-bf4fb3bd569c
+	k8s.io/kube-openapi v0.0.0-20200410145947-61e04a5be9a6
 	k8s.io/kubernetes v1.18.2
 	k8s.io/metrics v0.0.0
-	sigs.k8s.io/controller-runtime v0.6.0
+	sigs.k8s.io/controller-runtime v0.6.2
 )
 
 // Pinned to kubernetes-1.18.2
 replace (
```

Comment:
As you removed the `operator-sdk` dependency, you should be able to remove the replace section entirely

Reply:
unfortunately no, I pull in `k8s.io/kubernetes`, so there are some indirect deps I need:
```
go: k8s.io/kubernetes@v1.18.2 requires
	k8s.io/sample-apiserver@v0.0.0: reading k8s.io/sample-apiserver/go.mod at revision v0.0.0: unknown revision v0.0.0
[...]
go: k8s.io/kubernetes@v1.18.2 requires
	k8s.io/legacy-cloud-providers@v0.0.0: reading k8s.io/legacy-cloud-providers/go.mod at revision v0.0.0: unknown revision v0.0.0
```

This is something I had identified however, and we have a card in the backlog to move away from k8s/k8s to only get the versioned client. That should help with the deps.